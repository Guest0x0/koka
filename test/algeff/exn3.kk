effect abort
  ctl abort() : a

fun main() : total int
  with ctl abort() 42
  abort()
  0