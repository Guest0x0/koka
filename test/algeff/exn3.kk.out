42
 
algeff/exn3/abort: forall<a> () -> abort a
algeff/exn3/main: () -> int